common/src/main/java/net/william278/husksync/config/Settings.java

```diff
@@ -105,6 +105,21 @@ public static class DatabaseCredentials {
                     "useUnicode=true", "characterEncoding=UTF-8");
         }
 
+        @Comment("Specify credentials here for your MONGO database (This is only required if you have set the database type to \"MONGO\")")
+        private MongoDatabaseCredentials mongoCredentials = new MongoDatabaseCredentials();
```

Comment:
Given that these are largely duplicate from the normal DatabaseCredentials and I think this could cause confusion, why not just add a `mongoAuthDb` param to the existing DatabaseCredentials object, and comment that the connection params are only used for MYSQL/MARIADB?

Reply:
The reason i separated them was to minimise confusion as mongo is (imo) harder for the end user to setup than mysql is, but i see the point, will change that